### PR TITLE
PR for complete UI

### PR DESCRIPTION
**Pull request by MeghaSharma21**
@tgr, @slaporte please review.
The improvements mentioned for the tool (in the feedback mail) will be addressed in the next PR.

### CODE REVIEW

File: src/user_summary/static/user_summary/js/main.js [hunk 1]
```diff
@@ -0,0 +1,909 @@
...
+                success: function (data) {
+            google.charts.setOnLoadCallback(function () {
+                drawPieChart(data.spread_over_projects_data);
+                drawCalendarChart(data.created_activity_chart_data,
+                        'Articles Created', 'articles-created');
+                drawCalendarChart(data.edits_activity_chart_data,
+                        'Articles Edited', 'articles-edited');
+            });
+        },
+                error: function () {
```
Comment (tgr) [inline src/user_summary/static/user_summary/js/main.js:604]: This seems to be repeated a couple times, maybe it should be a separate function.

File: src/user_summary/static/user_summary/js/main.js [hunk 2]
```diff
@@ -0,0 +1,909 @@
...
+    });
+
+
+    //Whenever time filter is changed for impact, change the graph
+    $('input.impact-time-filter').on('change', function () {
+        $('input.impact-time-filter').not(this).prop('checked', false);
+        $.ajax({url: "/outreachy-wikicv/load-graphs/",
+                type: "get",
+                data: {
+            username: String(username),
```
Comment (tgr) [inline src/user_summary/static/user_summary/js/main.js:621]: I don't think explicit casts have any effect, HTTP request data is always a string.

**Review by tgr — APPROVED** [review]
Looks good at a glance, although I can't follow the DOM manipulation code in my head; I'll test it after it is deployed.

File: src/user_summary/static/user_summary/js/main.js (hunk 1 above)
Comment (MeghaSharma21) [reply src/user_summary/static/user_summary/js/main.js:604]: Okay.

File: src/user_summary/static/user_summary/js/main.js (hunk 2 above)
Comment (MeghaSharma21) [reply src/user_summary/static/user_summary/js/main.js:621]: Okay.

**Pull request merged**